src/bitmap.h

```diff
@@ -362,6 +350,37 @@ class Bitmap {
 	 */
 	void WaverBlit(int x, int y, double zoom_x, double zoom_y, Bitmap const& src, Rect const& src_rect, int depth, double phase, Opacity const& opacity);
 
+	/**
+	 * Blits source bitmap with transformation and opacity scaling.
+	 *
+	 * @param fwd forward (src->dst) transformation matrix.
```

Comment:
comment is outdated

Reply:
Actually I meant the param list (fwd), not the text above, but good catch.

Reply:
Oops, fixed that one too now.